pkg/services/ngalert/notifier/channels/template_data.go

```diff
@@ -89,6 +89,25 @@ func extendAlert(alert template.Alert, externalURL string, logger log.Logger) *E
 			u.RawQuery = "viewPanel=" + panelId
 			extended.PanelURL = u.String()
 		}
+
+		generatorUrl, gErr := url.Parse(extended.GeneratorURL)
```

Comment:
You can just use `err` here, it's OK to re-use it

```suggestion
		generatorUrl, err := url.Parse(extended.GeneratorURL)
```

Reply:
Hi, The suggested change has been made to re-use the same `err` variable